### ISSUE TITLE
[Question] Is there a way to disable the 5.25" drives?

### ISSUE THREAD

**Issue by gdanko** [david-schmidt/gsport#19]
If I open an application and exit that application, the Finder wants to scan both 5.25" drives and it's a bit of a hassle waiting.

**Comment by david-schmidt** [comment]
Nope, the "hardware" is always present - so it'll get scanned whenever GSOS wants to re-check for a disk swap.

**Comment by pacpir** [comment]
My suggestion would be just to go into control panel and change it to my
disk for Slot 6.
Perry

**Comment by NJRoadfan** [comment]
In GS/OS, you can set Finder to disable scanning the 5.25" drives (Special...., Preferences, uncheck "Check 5.25 drives on Finder Start"). Alternatively, you can disable the driver. Open up System/Drivers, click on "AppleDisk5.25", then "Get Info..." and check off the "Inactive" checkbox.
